packages/radix/src/components/Link.tsx

```diff
@@ -11,14 +17,56 @@ export const Link = React.forwardRef<HTMLAnchorElement, LinkProps>((props, forwa
     styleConfig={{
       base: {
         link: {
-          normal: {
-            color: theme.colors.blue700,
+          hover: {
+            textDecoration: 'underline',
           },
-          focus: {
-            boxShadow: `0 0 0 2px ${theme.colors.blue500}`,
+        },
+      },
+      variants: {
+        variant: {
+          ghost: {},
```

Comment:
ghost is missing a focus state

Reply:
Yeah, the idea of a `ghost` variant is that it inherits the color property. So if it's inside a red paragraph, it'll be red. 

Not sure which focus state to add in this case, since we cant inherit box shadow, as we can with text decoration color. 

Maybe just show an underline on focus too?

Reply:
Maybe, or you could have the same style as the other but set a box shadow using currentColor?

Reply:
Yeah that sounds like a decent fallback. For some reason I completely forgot about `currentColor` 😂

Reply:
Hmm 
![image](https://user-images.githubusercontent.com/372831/77168158-a3ed0380-6ab7-11ea-8153-31be8fcacf64.png)

Not sure that works in practice. It's too much. 

Whenever we use box shadow for focus, the color is also lighter than the text.

Reply:
Then I'd say do something similar to the fade one, like a light color or even a black with low opacity.